webapp/src/ts/app.component.html

```diff
@@ -12,6 +12,7 @@
   </div>
 
   <mm-privacy-policy [overlay]="true" *ngIf="showPrivacyPolicy && !privacyPolicyAccepted"></mm-privacy-policy>
+  <mm-sidebar-menu [canLogOut]="canLogOut"/>
```

Comment:
When you close the Sidebar menu the hamburger menu retains this slight background until you click elsewhere. Is that something easy to remove ? 
<img width="354" alt="Screenshot 2024-09-02 at 14 03 43" src="https://github.com/user-attachments/assets/6d765a52-50cc-432a-ac4a-8c67a74467a1">

Reply:
Unfortunately no. That's an effect that comes with Mat Button, I tried 
- [disableRipple]="true"
- event.preventDefault();
- event.stopPropagation();
- event.target.blur();
But nothing seems to remove that.

Reply:
I'm going to leave it like this for now. I haven't found a solution yet in 2h digging.

Reply:
done